var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -40,7 +44,14 @@ def url_for_version(self, version):
     def configure_args(self):
         spec = self.spec
         # have meme build its own versions of libxml2/libxslt, see #6736
-        args = ['--enable-build-libxml2', '--enable-build-libxslt']
+        #args = ['--enable-build-libxml2', '--enable-build-libxslt']
+        args = []
```

Comment:
Remove the commented line and fix the comment above. It also might be better to instead change `enable` to `disable` to be explicit.